tests/backends/base/test_base.py

```diff
@@ -55,6 +61,55 @@ def test_check_database_version_supported_with_none_as_database_version(self):
             connection.check_database_version_supported()
 
 
+class DatabaseWrapperLoggingTests(TransactionTestCase):
+    available_apps = []
+
+    @override_settings(DEBUG=True)
+    def test_commit_debug_log(self):
+        conn = connections[DEFAULT_DB_ALIAS]
+        with CaptureQueriesContext(conn):
+            with self.assertLogs("django.db.backends.base", "DEBUG") as cm:
+                with transaction.atomic():
+                    Person.objects.create(first_name="first", last_name="last")
+                    Person.objects.create(first_name="first", last_name="last2")
+
+                self.assertEqual(conn.queries_log[-1]["sql"], "COMMIT")
+                self.assertRegex(
+                    cm.output[0],
+                    r"DEBUG:django.db.backends.base:\(\d+.\d{3}\) "
+                    r"COMMIT; alias=default",
```

Comment:
I guess we could use `DEFAULT_DB_ALIAS` here as well

```suggestion
                    r"COMMIT; alias=%s" % DEFAULT_DB_ALIAS,
```

Reply:
fixed